src/wren_core.c

```diff
@@ -878,6 +878,24 @@ DEF_PRIMITIVE(num_cos)
   RETURN_NUM(cos(AS_NUM(args[0])));
 }
 
+DEF_PRIMITIVE(num_decimal)
+{
+  double value = AS_NUM(args[0]);
+  RETURN_NUM(value - (int32_t)value);
+}
+
+DEF_PRIMITIVE(num_deg)
+{
+  RETURN_NUM(floor(AS_NUM(args[0]) * 57.2957795130823208768));
+}
+
+DEF_PRIMITIVE(num_div)
+{
+  int32_t left = (int32_t)AS_NUM(args[0]);
+  int32_t right = (int32_t)AS_NUM(args[1]);
+  RETURN_NUM(left / right);
```

Comment:
In my plans this was meant to be a integer division, that is the same as
```
lsh.truncate / rhs.truncate
```
and not
```
(lhs / rhs).truncate
```
However, it now occurs to me that since both the division types can be easily obtained in Wren itself (now that we have the `truncate` method) perhaps the `div` method is useless.

Let's keep it simple, as usual, and leave it out. We would be forcing an (arbitrary) integer-division method when one may prefer the other and compute it by himself in Wren.

What do you think?